platform/src/main/resources/db/migration/V10__add_transaction_pending_trust_table.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE transaction_pending_trust (
+   id VARCHAR(255),
```

Comment:
Can we merge `V9__add_custody_table.sql` and V10__add_transaction_pending_trust_table.sql`?

Reply:
Fixed in #1067